pages/services/beta-storage/0.5.0-beta/index.md

```diff
@@ -2,7 +2,7 @@
 layout: layout.pug
 navigationTitle: DC/OS Storage Service 0.5.0
 title: DC/OS Storage Service 0.5.0
-menuWeight: 0
+menuWeight: -1
```

Comment:
Why do we have menu weight - 1 here? Does this mean - 0.5.0-beta version will not show up on docs?
If that is correct and 0.4.0-beta menu weight is 10? Should be we disabling/hiding the older version 0.4.0-beta?

Reply:
@imaxxs Yes, menu weight **-1** is assigned to pages/files that are not supported on https://docs.mesosphere.com/version-policy/#certified-packages-and-dcos-versions
![screen shot 2019-02-11 at 12 04 33 pm](https://user-images.githubusercontent.com/36425832/52590306-d11bac80-2df5-11e9-9f47-e905edb859db.png)

As per the above supported versions, I've assigned the following menu weights:
DC/OS Storage Service 0.5.0 - **-1**